scripts/webpack/bundle.js

```diff
@@ -110,6 +108,17 @@ module.exports = (env = {}) => {
     config.output.filename = 'dist/dist.dev.js';
     // Disable transpilation
     config.module.rules = [];
+  } else if (env.es5) {
+    config.output.filename = 'dist/dist.es5.min.js';
+    config.module.rules = [
+      {
+         // Compile ES2015 using babel
+         test: /\.js$/,
+         loader: 'babel-loader',
+         include: /src/,
+         options: ES5_BABEL_CONFIG
+      }
+    ];
```

Comment:
We should be able to return an array of configurations at the bottom here.

Reply:
DOne